internal/pkg/ccloudv2/flink_gateway.go

```diff
@@ -37,6 +37,20 @@ func NewFlinkGatewayClient(url, userAgent string, unsafeTrace bool, authToken st
 	}
 }
 
+func (c *FlinkGatewayClient) SetAuthToken(authToken string) {
+	if c == nil {
```

Comment:
I don't think the client should ever be `nil`. We always fetch the client or encounter an error first. How about making this variable public and omitting the getter/setter?

Reply:
Yeah sounds good, made it public now!